hotfn/http/request.py

```diff
@@ -102,15 +111,28 @@ def parse_raw_request(self):
             major, minor = major_minor.pop(), "0"
 
         # todo: parse query parameters carefully
+        params = parse_query_params(path)
+
         if headers.get('transfer-encoding', 'identity') == 'chunked':
             self.body_stream = ChunkedStream(self.stream)
-        else:
+        elif 'content-length' in headers:
```

Comment:
this would fail for async because content-length is not specified, see how we get content length few lines below.

Do following:
```
elif headers.get("content-length", header.get("fn_header_content_length", 0)):
```

Reply:
Those prefixes are stripped out now